flink/src/main/java/org/apache/iceberg/flink/source/RowDataFileScanTaskReader.java

```diff
@@ -48,7 +48,6 @@
 
 @Internal
 public class RowDataFileScanTaskReader implements FileScanTaskReader<RowData> {
-
```

Comment:
Looks like this file doesn't need to change. Can you remove this?

Reply:
will revert